modules/daicard/src/App.js

```diff
@@ -35,8 +35,8 @@ const { formatEther } = utils;
 
 const urls = {
   ethProviderUrl:
-    process.env.REACT_APP_ETH_URL_OVERRIDE || `${window.location.origin}/api/ethprovider`,
-  nodeUrl: process.env.REACT_APP_NODE_URL_OVERRIDE || `${window.location.origin}`,
+    process.env.REACT_APP_ETH_URL_OVERRIDE || `${window.location.origin}/ethprovider`,
```

Comment:
We'll need to fix this for multichain.

Reply:
Well that'll be a larger overhaul to add multichain support to the daicard so is pretty far outside the scope of this PR..
However, Indra's proxy *should* expose providers for each of it's chains.. I'll look into that